doc/api/esm.md

```diff
@@ -16,19 +16,19 @@ code for reuse. Modules are defined using a variety of [`import`][] and
 The following example of an ES module exports a function:
```

Comment:
I hate to be controversial, but I would prefer to simply recommend `"type": "module"` as the standard approach:

> In order to use modules with `.js` extensions, ensure you have a package.json file with `"type": "module"` set:

```js
// package.json
{
  "name": "app",
  "...",
  "type": "module"
}
```

Reply:
It sounds like you should then make a pr that makes `type: module` the default.

Reply:
Ok thanks for hearing me out :) I've created https://github.com/nodejs/node/pull/33409.